src/goto-instrument/goto_instrument_parse_options.cpp

```diff
@@ -1851,6 +1851,7 @@ void goto_instrument_parse_optionst::help()
     "Other options:\n"
     " --version                    show version and exit\n"
     HELP_FLUSH
+    " --xml                        output files in XML where supported\n"
```

Comment:
Is it possible to add some documentation in the `docs/` folder for this? I wasn't aware of the difference between `--xml` and `--xml-ui`

Reply:
It is but...

A. The current state of the `goto-instrument` documentation ( https://github.com/diffblue/cbmc/blob/develop/doc/cprover-manual/goto-instrument.md ), apart from function body generation and contracts is pretty abysmal.  I do want to improve it  but it is after "tidying up, figuring out and testing" the options on my TODO list.

B. I am not sure that it should even be an option or called that.  At the moment I am going with preserving functionality but I am also building a set of "I think this would be sensible" changes.  As it stands it is a formerly undocumented option that works with only two of the flags.  I do wonder if it would make more sense if it followed the same convention as `goto-analyzer`.  Or maybe it would be better to move all of the functionality over to `goto-analyzer`?